pkgs/applications/video/frigate/default.nix

```diff
@@ -52,6 +52,14 @@ let
     url = "https://github.com/openvinotoolkit/open_model_zoo/raw/master/data/dataset_classes/coco_91cl_bkgr.txt";
     hash = "sha256-5Cj2vEiWR8Z9d2xBmVoLZuNRv4UOuxHSGZQWTJorXUQ=";
   };
+
+  # Audio model
+  # https://github.com/blakeblackshear/frigate/blob/v0.13.2/docker/main/Dockerfile#L104
+  audio_model = fetchurl {
+    url = "https://www.kaggle.com/api/v1/models/google/yamnet/tfLite/classification-tflite/1/download";
```

Comment:
Where does this URL come from? The Dockerfile points to https://tfhub.dev/google/lite-model/yamnet/classification/tflite/1?lite-format=tflite.